internal/server/models/forgotdomain.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import (
+	"time"
+)
+
+type ForgottenDomain struct {
+	OrganizationName   string
+	OrganizationDomain string
+	LastSeenAt         time.Time
```

Comment:
Seems like `LastSeenAt` isn't used anywhere, let's leave it out for now

Reply:
I ended up using this in the latest change as I think it's somewhat important to give you an idea about when you last signed-in.